schema_based_on_diagram.sql

```diff
@@ -0,0 +1,64 @@
+----- Create Clinic Database -----
+CREATE DATABASE clinic;
+
+----- patients table -----
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  date_of_birth DATE NOT NULL
+);
+
+----- medical histories table -----
+CREATE TABLE medical_histories(
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP NOT NULL,
+  patient_id INT NOT NULL,
+  status VARCHAR(255) NOT NULL,
+  CONSTRAINT fk_patient_constraint FOREIGN KEY(patient_id) REFERENCES patients(id)
+);
+
+----- invoices table -----
+CREATE TABLE invoices(
+  id SERIAL PRIMARY KEY,
+  total_amount DECIMAL(5, 2),
+  generated_at TIMESTAMP NOT NULL,
+  payed_time TIMESTAMP NOT NULL,
```

Comment:
- [ ] The data entry in the ERD diagram is called `payed_at` & not `payed_time`. Kindly change this entry if you can as it might cause problems in the future Milestones or when entering data.